### ISSUE TITLE
I have trained 80 epochs but it's slow in TITANX. Is that normal?

### ISSUE THREAD

**Issue by zaie** [feichtenhofer/twostreamfusion#33]
14 hours per epoch on titan X, nvidia-smi shows that GPU is used. And the results is not very good. The loss is like that:
![image](https://cloud.githubusercontent.com/assets/17739876/26310363/18553886-3f33-11e7-97f4-1bc8b24951a1.png)
 Is that normal?

**Comment by feichtenhofer** [comment]
You've trained for way too long. Each training epoch is actually 10 epochs through the dataset (see opts.train.epochFactor = 10;). You can stop training after the last learning rate reduction (i.e. epoch 5).
